Sources/Components/CollapseView/CollapseView.swift

```diff
@@ -0,0 +1,188 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+public protocol CollapseViewDelegate: AnyObject {
+    func didExpand(_ view: CollapseView)
```

Comment:
How you considered `collapseViewDidExpand()` / `collapseViewDidCollapse()` instead? I think this would be more in line with general delegate naming-conventions. :-)